src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -217,9 +205,51 @@ public void onTraversalStart() {
                 logger.info("Sample id " + sampleId + " was detected as already loaded, exiting successfully.");
                 System.exit(0);
             } else if (state == LoadStatus.LoadState.PARTIAL) {
```

Comment:
Do you want this to be PARTIAL || NONE?  What happens if we've never attempted to load this sample before (which is the most common case by far)